nnlo/train/data.py

```diff
@@ -90,25 +93,28 @@ def set_file_names(self, file_names):
                 relocate = goes_to+'/'+fn.split('/')[-1]
                 if not os.path.isfile( relocate ):
                     logging.info("copying %s to %s", fn , relocate)
-                    if (self.use_s3):
-                       if os.system('s3cmd get s3://gan-bucket/%s %s'%( fn ,relocate))==0:
-                          relocated.append( relocate )
-                       else:
-                          logging.info("was unable to copy the file s3://ganbucket/%s to %s",fn,relocate)
-                          relocated.append( fn ) ## use the initial one
-                    else:   
-                       if os.system('cp %s %s'%( fn ,relocate))==0:
-                          relocated.append( relocate )
-                       else:
-                          logging.info("was enable to copy the file %s to %s",fn,relocate)
-                          relocated.append( fn ) ## use the initial one
+                    if self.copy_command:
+                        cmd = self.copy_command.format( fn, relocate )
+                        if os.system(cmd)==0:
+                            relocated.append( relocate )
+                        else:
+                            logging.error("was unable to copy the file with {}".format( cmd ))
+                            relocated.append( fn ) ## use the initial one
+                    else:
+                        ## admitedly, this is redundant, as copy_command could default to "cp {} {}"
```

Comment:
We can set the default `copy_command` in `Data`s `__init__` to get rid of this if statement.

Reply:
yup